### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by andela-iokonkwo** [andela-cdaniel/Group_Discuss#1]
- Is both the tasker and taskee categorized according to levels (regular, gold and platinum) ?
- Is the taskee also paying a monthly subscription fee?

**Comment by andela-oojewale** [comment]
Hello Ikem,

Prior to our discussion, we have met and we have answered the questions you raised on the current model of Workdey.

The Tasker - The person who is in charge of giving out tasks to be executed.
The Taskee - The person who is in charge of executing specific tasks.

Both the Tasker and the Taskee are categorized according to levels.
The gold level of the Tasker is also based on verification and until successful completion of this verification process, the gold level will not be reflected on the Tasker's profile.

The platinum level of the Tasker is automated based on the amount of time spent on the site and the amount of tasks given out, while that of the Taskee is based on amount of time spent on the site, the number of tasks taken and positive reviews from Taskers.

The Taskees are also entitled to three months of trial service. However, they will pay a monthly subscription fee after this trial duration of this three months.

**Comment by andela-iokonkwo** [comment]
Is the verification amount for both taskee and taskers the  same.

**Comment by andela-oojewale** [comment]
They are not the same. The Taskee's price should be less than that of the Tasker.

**Comment by andela-iokonkwo** [comment]
Shouldn't  the taskees be exempted from the monthly subscription?

**Comment by andela-atemitope** [comment]
no, they shouldn't.. they are paying for our service..to use our platform to market their skills
